### ISSUE TITLE
Build Item View

### ISSUE THREAD

**Issue by ashatat** [ashatat/e-commerce#8]

**Comment by ashatat** [comment]
@FarahZaqout @amusameh @ishak52 @RamyAlshurafa 

Don't you think this is so much? a lot of details for an epic?
maybe we should move the endpoints details on each detailed issue?

**Comment by FarahZaqout** [comment]
Agreed. Once we start detailing, we will cut each issue's details and move it there.
This is some great experimenting. We should be able to guide FACG5 a lot better in the next phase as a side effect.
